packages/blocks/src/__internal__/utils/components.ts

```diff
@@ -1,11 +1,14 @@
 /* eslint-disable lit/binding-positions, lit/no-invalid-html */
 import '../../components/loader.js';
 
-import { BLOCK_CHILDREN_CONTAINER_PADDING_LEFT } from '@blocksuite/global/config';
+import {
+  BLOCK_CHILDREN_CONTAINER_PADDING_LEFT,
+  BLOCK_ID_ATTR,
+} from '@blocksuite/global/config';
 import type { BaseBlockModel } from '@blocksuite/store';
 import { matchFlavours } from '@blocksuite/store';
 import { repeat } from 'lit/directives/repeat.js';
-import { html } from 'lit/static-html.js';
+import { html, unsafeStatic } from 'lit/static-html.js';
```

Comment:
Is it possible to replace this with the `literal` directive?

Reply:
> Error: Value passed to 'literal' function must be a 'literal' result: data-block-id. Use 'unsafeStatic' to pass non-literal values, but
            take care to ensure page security.

It's not work, because tags a string literal so that it behaves like part of the **static template strings** instead of a dynamic value. [ref](https://lit.dev/docs/api/static-html/#literal)

At this location, the unsafe template we're using only covers static strings inside the code, so it's security. We could directly use the string "data-block-id", but this would lose the reference to `BLOCK_ID_ATTR`.
```jsx
      return html`
        <${model.tag}
          .model=${model}
          .host=${host}
          data-block-id=${model.id}
        ></${model.tag}>
      `;
```